firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Unreleased
+- [changed] SSL initialization now happens on a separate thread, which should
```

Comment:
"should improve" makes it sound like we didn't test it. How about "which reduces the time taken to produce the first query results." ?

Reply:
That was not totally unintended. There is some overhead that I assume is much lower than blocking on GMSCore, but I only tested this change on the one device I have. With that being, I don't mind being more assertive and updated the message.